.gitignore

```diff
@@ -109,3 +109,6 @@ public/bundle.js.map
 
 #env variables
 .env
+
+#package-lock bc it's annoying as hell
```

Comment:
lol

Reply:
i think every time you pull from main you just have to 'npm i' to install all the packages which would automatically created the package-lock file